src/models/notifications.js

```diff
@@ -7,27 +7,29 @@ const UnauthorizedError = require('five-bells-shared/errors/unauthorized-error')
 const Notification = require('./db/notification').Notification
 const Transfer = require('./db/transfer').Transfer
 const Fulfillment = require('./db/conditionFulfillment').ConditionFulfillment
+const Subscription = require('./db/subscription').Subscription
 const subscriptionUtils = require('../lib/subscriptionUtils')
 
 function * getNotification (subscriptionId, notificationId, requestingUser) {
   log.debug('fetching notification ID ' + notificationId)
-  const notification = yield Notification.findOne({
-    where: {
-      id: notificationId
-    },
-    include: [{ all: true }]})
+  const notification = yield Notification.findById(notificationId)
```

Comment:
We may be able to do this more easily with a join (so we don't need two more queries to get the subscription and transfer), but I think it's fine to leave as is, for this PR.